packages/fxa-auth-server/scripts/create-mock-iap-subscriptions.ts

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import program from 'commander';
+
+import { setupProcessingTaskObjects } from '../lib/payments/processing-tasks-setup';
+
+const pckg = require('../package.json');
+import * as mockIapSubscriptions from './create-mock-iap-subscriptions/mock-iap-subscriptions.json';
+import { MockIapSubscriptionManager } from './create-mock-iap-subscriptions/mock-iap-subscription-manager';
+
+export async function init() {
+  program
+    .version(pckg.version)
+    .option('-c, --config [config]', 'Configuration to use. Ex. dev')
```

Comment:
We only use the emulator locally, so do we need to have an env option (see also [this comment](https://github.com/mozilla/fxa/pull/14227/files#r992867901))? Also, with this line, it looks like we're ignoring the `NODE_ENV` passed into the npm run script command?

Reply:
Yep this is pretty much useless. Had preserved it since it seemed to be a common pattern in other scripts but there's no reason for it to be there.